app/components/date-time-picker.js

```diff
@@ -1,22 +1,29 @@
 import Component from '@glimmer/component';
 import { action } from "@ember/object";
 import { inject as service } from '@ember/service';
+import { tracked } from '@glimmer/tracking';
 
 export default class DateTimePicker extends Component{
   @service intl;
-  date;
-  hours;
-  minutes;
+  @tracked date;
 
   constructor() {
     super(...arguments);
     if(this.args.value) {
       this.date = new Date(this.args.value);
-      this.hours = this.date.getHours();
-      this.minutes = this.date.getMinutes();
+    } else {
+      this.date = new Date();
```

Comment:
The current version always creates a new date when one isn't passed in, so it will always default to the current time in that case. This is needed because the time inputs are now derived from the date.

Would it make sense to not set a default value and disable the time inputs instead? The user would need to select a date first if non is provided? I think that might make more sense UX wise.